### PR TITLE
nits

### PR DESCRIPTION
**Pull request by AkihiroSuda**
```
00891cd limactl: improve help output
7591b14 alpine: start networking earlier
021556e hostagent: propagate --debug
ac46219 README.md: update the status of the QEMU patches
f81883a pkg/hostagent/api -> pkg/hostagent/events
```

### CODE REVIEW

File: pkg/cidata/cidata.TEMPLATE.d/boot/10-alpine-prep.sh [hunk 1]
```diff
@@ -32,3 +32,6 @@ rc-service cgroups start
 # `limactl stop` tells acpid to powerdown
 rc-update add acpid
 rc-service acpid start
+
+# enable networking explicitly here for apk
+rc-service networking start
```
Comment (AkihiroSuda) [inline pkg/cidata/cidata.TEMPLATE.d/boot/10-alpine-prep.sh:37]: cc @jandubois

Comment (jandubois) [reply pkg/cidata/cidata.TEMPLATE.d/boot/10-alpine-prep.sh:37]: This is really a bug in `lima-init`, as it works correctly with `cloud-init`. So I think it should just be fixed in the lima-init openrc script instead.

File: cmd/limactl/main.go [hunk 2]
```diff
@@ -23,10 +25,30 @@ func main() {
...
+  $ limactl start
+
+  Open a shell:
+  $ lima
+
+  Run a container:
+  $ lima nerdctl run -d --name nginx -p 8080:80 nginx:alpine
+
+  Stop the default instance:
+  $ limactl stop
```
Comment (jandubois) [inline cmd/limactl/main.go:49]: This does not work right now, as the stop command has `cobra.ExactArgs(1)`. So you have to write `limactl stop ""`. This should be fixed by using `cobra.MaximumNArgs(1)` instead.

File: pkg/start/start.go [hunk 3]
```diff
@@ -72,10 +72,17 @@ func Start(ctx context.Context, inst *store.Instance) error {
...
+	}
+
 	haCmd := exec.CommandContext(ctx, self,
-		"hostagent",
-		"--pidfile", haPIDPath,
-		inst.Name)
+		append(debugFlags, []string{
+			"hostagent",
+			"--pidfile", haPIDPath,
+			inst.Name}...)...)
```
Comment (jandubois) [inline pkg/start/start.go:84]: I find the `}...)...)` construct not very readable. Wouldn't it be better to just build up an `args` list upfront and then call:

```
	var args []string
	if logrus.GetLevel() >= logrus.DebugLevel {
		args = append(args, "--debug")
	}
	args = append(args,
		"hostagent",
		"--pidfile", haPIDPath,
		inst.Name)
	haCmd := exec.CommandContext(ctx, self, args...)
```

File: pkg/cidata/cidata.TEMPLATE.d/boot/10-alpine-prep.sh (hunk 1 above)
Comment (jandubois) [reply pkg/cidata/cidata.TEMPLATE.d/boot/10-alpine-prep.sh:37]: I've implemented the fix in https://github.com/lima-vm/alpine-lima/pull/19

Comment (AkihiroSuda) [reply pkg/cidata/cidata.TEMPLATE.d/boot/10-alpine-prep.sh]: dropped

File: cmd/limactl/main.go (hunk 2 above)
Comment (AkihiroSuda) [reply cmd/limactl/main.go:49]: fixed

File: pkg/start/start.go (hunk 3 above)
Comment (AkihiroSuda) [reply pkg/start/start.go]: fixed

**Review by jandubois — APPROVED** [review]

**Pull request merged**